### ISSUE TITLE
4.28 nightly build broken

### ISSUE THREAD

**Issue by iloveeclipse** [eclipse-platform/eclipse.platform.releng.aggregator#1026]
See https://ci.eclipse.org/releng/job/Builds/job/I-build-4.28/

- https://ci.eclipse.org/releng/job/Builds/job/I-build-4.28/67/
- https://ci.eclipse.org/releng/job/Builds/job/I-build-4.28/68/

Looks like it stuck at "Create Base builder stage" for minutes instead of few seconds:
![image](https://user-images.githubusercontent.com/964108/232290589-43944355-5382-48ee-8f1c-5c5eceed0088.png)

Could be related to latest docker image build https://ci.eclipse.org/releng/job/Build-Docker-images/515/console that was executed before. **Last successful** build was https://download.eclipse.org/eclipse/downloads/drops4/I20230414-1800.

**Comment by merks** [comment]
I've seen a number of builds hanging and aborted, or I abort them because they make no progress for way too long, but then the next build works fine.  I don't see a pattern to this behavior.

**Comment by iloveeclipse** [comment]
> I've seen a number of builds hanging and aborted, or I abort them because they make no progress for way too long, but then the next build works fine. I don't see a pattern to this behavior.

I haven't seen **such** build fails in SDK recently. 
However, second build hangs again on same stage, and this looks like a pattern for me.

**Comment by merks** [comment]
I would just kill #68 and start a new one and hope for the best.  At least if it consistently gets stuck there we'd have something to track down...

**Comment by iloveeclipse** [comment]
> I would just kill #68 and start a new one and hope for the best. At least if it consistently gets stuck there we'd have something to track down...

Build 68 was started by me already, but OK.

**Comment by iloveeclipse** [comment]
Started https://ci.eclipse.org/releng/job/Builds/job/I-build-4.28/69/.

**Comment by merks** [comment]
This one is doing better!  This is really a very annoying problem.  😡

**Comment by laeubi** [comment]
@merks without any deeper investigation, this reminds me on the recent P2 build that also got stuck "forever" lately so maybe an infrastructure issue?

- https://github.com/eclipse-equinox/p2/pull/245

**Comment by merks** [comment]
@laeubi 

Exactly.  It also happened today with a releng build and a p2 build so not an isolated incident...

**Comment by sravanlakkimsetti** [comment]
One clarification the nightly build do not use docker images from https://ci.eclipse.org/releng/job/Build-Docker-images/515/console
